packages/discvr-utils/package.json

```diff
@@ -1,6 +1,9 @@
 {
     "name": "discvr-utils",
     "version": "0.0.0",
+    "scripts": {
+        "test": "echo \"Testing utils\""
+    },
     "dependencies": {
```

Comment:
For all the reports, we should think carefully about how strictly we declare dependencies. Here we are linked to a very specific reactJS. Especially for the utils package, that is probably not true. We expect this to be a pretty low-level package, and therefore it would make it easier for packages that depends on discvr-utlis if this was not pegged to such a specific react if it doesnt need to be.

The other modules might have a genuine reason to depend on a specific React, or maybe they dont.

Reply:
I changed the version for react in utils to a '*' let me know if there is anything specific you would like it to be changed to.